### PR TITLE
Adding Brazilian exchange Walltime.

### PR DESCRIPTION
**Pull request by felipelalli**

### CODE REVIEW

**Comment by Mirobit** [comment]
Is there any update on their [statement](https://walltime.info/news/0002.txt) that they run Bitcoin Core?
If they continue to do, they should be added.

**Comment by felipelalli** [comment]
@Mirobit we always will run Bitcoin Core. Do you need additional confirmation? I'm the Walltime CEO and you can verify it by seeing the GPG signature in my commit. Thank you!

**Comment by gabrieldov** [comment]
I confirm, exchange exists and is operated by @felipelalli as stated above.

**Comment by AugustoResende** [comment]
Me too.

File: _templates/exchanges.html [hunk 1]
```diff
@@ -65,6 +65,7 @@ <h3 id="brazil"><img src="/img/flags/BR.png" alt="Brazilian flag">Brazil</h3>
       <p>
         <a href="https://foxbit.com.br/">Foxbit</a><br>
         <a href="https://www.mercadobitcoin.com.br/">Mercado Bitcoin</a>
```
Comment (wbnns) [inline _templates/exchanges.html]: A line break (`<br>`) is missing after Mercado Bitcoin. Without it, the listings will break - could you please fix?

**Comment by felipelalli** [comment]
@wbnns I'm sorry, I just added and pushed force to fix.

**Comment by wbnns** [comment]
@felipelalli No problem, thanks!

**Pull request merged**